types/parse/index.d.ts

```diff
@@ -830,11 +836,31 @@ subscription.on('close', () => {});
         function beforeFind(arg1: any, func?: (request: BeforeFindRequest) => Promise<Query> | Query): void;
         function afterFind(arg1: any, func?: (request: AfterFindRequest) => Promise<any> | any): void;
         function define(name: string, func?: (request: FunctionRequest) => Promise<any> | any): void;
+        /**
+         * Gets data for the current set of cloud jobs.
+         * @returns A promise that will be resolved with the result of the function.
+         */
+        function getJobsData(): Promise<any>;
```

Comment:
```suggestion
        function getJobsData(options?: RequestOptions): Promise<any>;
```

Reply:
Thanks for the feedback. Where did you get this params from? they're not showed in the [docs](https://parseplatform.org/Parse-SDK-JS/api/2.7.0/Parse.Cloud.html#.getJobsData).

Reply:
You're welcome! Again, I've got [those from source code](https://github.com/parse-community/Parse-SDK-JS/blob/05e8c79e17c8d7f30280303c41848dcd15b6d5cb/src/Cloud.js#L143).

Reply:
Got it. I think the desired function is [this one ](https://github.com/parse-community/Parse-SDK-JS/blob/719e98ff74ee99b820fcb8e6c37f32695d07e307/src/Cloud.js#L71) on the same file, but takes no arguments.
However it ruturns a `Promise<Parse.Object>`.